### PR TITLE
Make valgrind test verbose

### PR DESCRIPTION
**Pull request by foursixnine**
VR: https://openqa.suse.de/tests/10716187#

### CODE REVIEW

File: tests/console/valgrind.pm [hunk 1]
```diff
@@ -44,7 +44,7 @@ sub run {
     assert_script_run 'cd /var/tmp';
     assert_script_run 'curl -v -o valgrind-test.c ' . data_url('valgrind/valgrind-test.c');
     assert_script_run 'curl -v -o valgrind-test.sh ' . data_url('valgrind/valgrind-test.sh');
-    assert_script_run 'bash -e valgrind-test.sh';
+    assert_script_run 'bash -e -x valgrind-test.sh';
```
Comment (rfan1) [inline tests/console/valgrind.pm:47]: Can we modify the `valgrind-test.sh` script and add `-v` option for valgrind command?

Comment (grisu48) [reply tests/console/valgrind.pm:48]:
```suggestion
    assert_script_run 'bash -ex valgrind-test.sh';
```

File: data/valgrind/valgrind-test.sh [hunk 2]
```diff
@@ -7,6 +7,7 @@
 # Regression test for valgrind
 # Maintainer: Felix Niederwanger <felix.niederwanger@suse.de>
 
+set -x
```
Comment (grisu48) [inline data/valgrind/valgrind-test.sh:10]: I would rather do this in the shebang:

```
#!/bin/bash -x
```

File: data/valgrind/valgrind-test.sh [hunk 3]
```diff
@@ -7,6 +7,7 @@
 # Regression test for valgrind
 # Maintainer: Felix Niederwanger <felix.niederwanger@suse.de>
```
Comment (grisu48) [inline data/valgrind/valgrind-test.sh:8]: Could be moved to qe-core IMHO

**Review by grisu48 — COMMENTED** [review]
Some minor suggestions

**Comment by foursixnine** [comment]
@grisu48 done :) thanks!

File: data/valgrind/valgrind-test.sh [hunk 4]
```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/bash ex
 # SUSE's openQA tests
 #
 # Copyright 2020 SUSE LLC
```
Comment (grisu48) [inline data/valgrind/valgrind-test.sh:4]: This line and the maintainer in line 8 here as well. Sorry I could not select this before you edited the shebang.

**Review by grisu48 — APPROVED** [review]

**Review by grisu48 — APPROVED** [review]

**Review by rfan1 — APPROVED** [review]

**Pull request merged**